### PR TITLE
Add more tests for user

### PR DESCRIPTION
**Pull request by aanil**
* Add some more tests for user.py

### CODE REVIEW

File: tests/test_user_add.py [hunk 1]
```diff
@@ -41,7 +43,7 @@ def test_add_user_with_researcher(client):
         data=json.dumps(first_new_user),
         content_type="application/json",
     )
-    assert response.status == "403 FORBIDDEN"
+    response.status_code == http.HTTPStatus.FORBIDDEN
```
Comment (alneberg) [inline tests/test_user_add.py:46]: I think it's nice to keep the `assert` before the comparison?

File: tests/test_user_add.py [hunk 2]
```diff
@@ -137,7 +169,7 @@ def test_add_user_with_unitpersonnel_permission_denied(client):
         data=json.dumps(new_unit_admin),
         content_type="application/json",
     )
-    assert response.status == "403 FORBIDDEN"
+    response.status_code == http.HTTPStatus.FORBIDDEN
```
Comment (alneberg) [inline tests/test_user_add.py:172]: same here, keep the `assert`

File: tests/test_user_add.py [hunk 3]
```diff
@@ -241,7 +318,7 @@ def test_existing_user_change_ownership_same_permissions(client):
         data=json.dumps(user_same_ownership),
         content_type="application/json",
     )
-    assert response.status == "403 FORBIDDEN"
+    response.status_code == http.HTTPStatus.FORBIDDEN
```
Comment (alneberg) [inline tests/test_user_add.py:321]: here as well

File: tests/test_user_add.py [hunk 4]
```diff
@@ -255,4 +332,4 @@ def test_add_existing_user_with_unsuitable_role(client):
         data=json.dumps(user_with_unsuitable_role),
         content_type="application/json",
     )
-    assert response.status == "403 FORBIDDEN"
+    response.status_code == http.HTTPStatus.FORBIDDEN
```
Comment (alneberg) [inline tests/test_user_add.py:335]: And here

File: tests/test_user_info.py [hunk 5]
```diff
@@ -79,3 +79,16 @@ def test_get_info_superadmin_user(client):
...
+    """Show the amount of GB hours and the total cost for a unit"""
+    # Preliminary test - to be expanded later
+    response = client.get(
+        tests.DDSEndpoint.USAGE,
+        headers=tests.UserAuth(tests.USER_CREDENTIALS["unituser"]).token(client),
+        content_type="application/json",
+    )
+    assert response.status_code == http.HTTPStatus.OK
+    assert "project_usage" in response.json.keys()
+    assert "total_usage" in response.json.keys()
```
Comment (alneberg) [inline tests/test_user_info.py:94]: I know it's daunting, but we should perhaps test whether the data returned is valid? This is very nice for coverage though... 😂

File: tests/test_user_add.py (hunk 3 above)
Comment (aanil) [reply tests/test_user_add.py]: Done!

File: tests/test_user_add.py (hunk 4 above)
Comment (aanil) [reply tests/test_user_add.py]: Done!

File: tests/test_user_info.py (hunk 5 above)
Comment (aanil) [reply tests/test_user_info.py:94]: Yeah, but the returned data is all 0. I'm not sure if it should be mocked.

Comment (alneberg) [reply tests/test_user_info.py:94]: Ah, true. Might need to add some simple sizes to the files in the db then?

File: tests/test_user_add.py (hunk 1 above)
Comment (aanil) [reply tests/test_user_add.py]: 👍

File: tests/test_user_add.py (hunk 2 above)
Comment (aanil) [reply tests/test_user_add.py]: 👍

File: tests/test_user_info.py (hunk 5 above)
Comment (aanil) [reply tests/test_user_info.py:95]: The issue is that the upload time and the time at which the test is run is so close together that the function rounds it to zero.

**Review by inaod568 — APPROVED** [review]

**Pull request merged**